src/vaults/BalanceSheet.sol

```diff
@@ -325,4 +326,12 @@ contract BalanceSheet is Auth, Recoverable, IBalanceSheet, IBalanceSheetGatewayH
         queue.increase = 0;
         queue.decrease = 0;
     }
+
+    function _pricePoolPerAsset(PoolId poolId, ShareClassId scId) internal view returns (D18) {
+        return d18(TransientStorageLib.tloadUint128(keccak256(abi.encode("pricePoolPerAsset", poolId, scId))));
+    }
+
+    function _pricePoolPerShare(PoolId poolId, ShareClassId scId) internal view returns (D18) {
+        return d18(TransientStorageLib.tloadUint128(keccak256(abi.encode("pricePoolPerShare", poolId, scId))));
```

Comment:
CR

Same as above

Reply:
Just to be sure: We dont want to default here to the sync manager which might have a custom pPPS valuation right?

Reply:
Yes. The sync manager should use this override instead of the default pool manager price.

Reply:
Fixed

Reply:
My question was rather the opposite way: In my PR, we change the custom valuation to pPPS in the sync manager. 

So should BS fallback to syncRequests.pricePoolPerShare instead of poolmanager?

Reply:
@mustermeiszer wdyt?

Reply:
Merging for now, let's consider in a followup PR

Reply:
Answered above. I don't think the sync manager should be the default price provider. Too opaque.